### PR TITLE
Migrate remaining v-forms, v-simple-table

### PR DESCRIPTION
**Pull request by kelsos**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/components/common/SimpleTable.vue [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  };
+});
+</script>
+
+<template>
+  <div
+    :class="[
+      css.table,
+      {
+        [css.outlined]: variant == 'outlined'
```
Comment (tewshi) [inline frontend/app/src/components/common/SimpleTable.vue:37]: tripple `===` not necessary here right?

File: frontend/app/src/components/settings/data-security/ChangePassword.vue [hunk 2]
```diff
@@ -107,9 +100,9 @@ const change = async () => {
         :error-messages="toMessages(v$.newPasswordConfirm)"
         variant="outlined"
       />
-    </VForm>
+    </form>
 
-    <template #buttons>
+    <template #footer>
       <RuiButton
         depressed
```
Comment (tewshi) [inline frontend/app/src/components/settings/data-security/ChangePassword.vue:107]: we don't have this prop atm

File: frontend/app/src/components/common/SimpleTable.vue (hunk 1 above)
Comment (kelsos) [reply frontend/app/src/components/common/SimpleTable.vue:37]: it should be let me fix that.

File: frontend/app/src/components/settings/data-security/ChangePassword.vue (hunk 2 above)
Comment (kelsos) [reply frontend/app/src/components/settings/data-security/ChangePassword.vue:107]: hmm did I changed that after I tested the component?

Comment (kelsos) [reply frontend/app/src/components/settings/data-security/ChangePassword.vue:107]: Oh ok ok, I see there is a leftover property from pre-migration

**Review by tewshi — APPROVED** [review]
lgtm!

**Pull request merged**